src/styles/blocks/page.scss

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use `*` selector with caution due to its potential performance issues. In this case, it's used appropriately to set a global box-sizing, but keep in mind that for larger projects, you might want to apply `box-sizing: border-box;` more selectively.